### ISSUE TITLE
Integrations for #accessibility-testing

### ISSUE THREAD

**Issue by akirk** [Automattic/matrix-bots-wporg#14]
Accordingt to our records, the `#accessibility-testing` Slack channel on Making WordPress does not have any integrations that need to be migrated from Slack to Matrix.
